src/routes/legislative-process/+page.svelte

```diff
@@ -135,6 +177,29 @@
 		@apply text-text-primary mb-4;
 	}
 
+	h3 {
+		@apply text-text-primary mb-4;
+	}
+
+	table {
+		@apply bg-magenta-30;
+	}
+
+	:global(.bx--data-table th) {
```

Comment:
Should not use `:global` like this. It will apply everywhere because we don't know if other `DataTable` on this website want this custom style. I'm not sure if it is because you tried without `:global` and it didn't work. Or add some class specifically to this component on this page.

If it's not working at all, we might need to create our own table.

Reply:
sorry naka I forget to remove ka.